packages/engine/src/gltf/GLTFComponent.tsx

```diff
@@ -199,14 +200,19 @@ export const GLTFComponent = defineComponent({
   reactor: () => {
     const entity = useEntityContext()
     const gltfComponent = useComponent(entity, GLTFComponent)
+    const loaded = GLTFComponent.useSceneLoaded(entity)
+
+    useEffect(() => {
+      setVisibleComponent(entity, loaded)
```

Comment:
this will cause GLTFs invisible/hidden in the studio to become visible upon load

Reply:
Removed for now, functionality will be added back implicitly with the loader refactor